tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -70,6 +74,10 @@ describe('Eventing tests', function() {
     await waitForNamespace(mockNamespace);
   });
 
+  if (!isSKR) {
+    setGrafanaProxy();
+  }
```

Comment:
Why is it only enabled on OS? Please check if you could set/reset into the mocha hooks (before/after)?